lona/server.py

```diff
@@ -32,6 +40,8 @@
 http_logger = logging.getLogger('lona.server.http')
 websockets_logger = logging.getLogger('lona.server.websockets')
 
+_T = TypeVar('_T')
```

Comment:
Can we give this a better name?

Reply:
What about removing underscore?

Reply:
Aren't one character definitions an anti-pattern?

Reply:
In this case no. That's how mypy recommends define generics.
https://mypy.readthedocs.io/en/stable/generics.html

Reply:
And python docs as well https://docs.python.org/3/library/typing.html#generics

Reply:
@maratori Ah ok. I am ok with that then